install-mc-jars.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2015 M. Isuru Tharanga Chrishantha Perera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------
+# 2018-02-08 Modified to work with JDK 9
+# ----------------------------------------
+#
+# ----------------------------------------------------------------------------
+# Installing JMC Jars to a local repo
+# ----------------------------------------------------------------------------
+
+jmc_version=`echo $(basename $JAVA_HOME/lib/missioncontrol/plugins/com.oracle.jmc.*.jar) | sed 's/.*_\(.*\)\.jar/\1/'`
```

Comment:
This doesn't work with coreutils neither ubuntu 16.03 nor on bash for windows. Could be fixed by adding -a option basename which would probably break it mac, or by using com.oracle.jmc.common.*.jar as path.

Reply:
Added -a and it works OK on Mac with JDK 9